### DISCUSSION TITLE
Extra blank line with doc.toString

### DISCUSSION THREAD

**Discussion by srishina** [eemeli/yaml#324] [Q&A] [answered]
Hi, 

Many thanks for the excellent library.

I have a YAML document that I need to modify and later use toString to get the YAML representation. However for the following

`forecast: "How is the weather today?\nIs it good?"`

after toString

```
    forecast: "How is the weather today?
--> there is an extra blank line here
      Is it good?"
```
Is it expected? or is there an option that I can use to remove the extra blank line?

**Comment by eemeli** [comment, marked as answer]
In YAML, a double-quoted string needs to use `\n\n` to represent a newline, as a single `\n` would effectively get parsed as a space.

**Comment by srishina** [comment]
Thank you!

```
        const yamlData = `name: me\nforecast: "How is the weather today?\\nIs it good?"`;
        console.log("yamldata: ", yamlData)
        const yamlDoc = YAML.parseDocument(yamlData);
        console.log(yamlDoc.toString())
```

should result in:

`name: me
forecast: "How is the weather today?

  Is it good?"`

Is my understanding right?

**Reply by eemeli** [reply]
I'm not really sure what you're asking me. The library's output is valid YAML.

**Reply by srishina** [reply]
I think, I understand now. I was asking about the blank line , but I think as YAML needs `\n\n` it gets inserted in Document.toString function.
